demo/index.html

```diff
@@ -311,6 +311,22 @@ <h1>auro-radio</h1>
         </template>
       </demo-snippet>
 
+      <p>Group with onchange</p>
```

Comment:
I don't we need to add anything to the demo for this. Input and change events are already being logged in the console for every radio button on the demo page.